cpp/src/parquet/column_writer.h

```diff
@@ -175,6 +175,9 @@ class PARQUET_EXPORT ColumnWriter {
   /// total_bytes_written().
   virtual int64_t total_compressed_bytes_written() const = 0;
 
+  /// \brief Estimated size of the values that are not written to a page yet.
+  virtual int64_t EstimatedBufferedValueBytes() const = 0;
```

Comment:
It is unrelated, but the naming looks weird after relocation...

Reply:
+1